wiki/Modding/Timestamp/id.md

```diff
@@ -0,0 +1,25 @@
+---
+tags:
+  - edit link
+  - timestamps
+---
+
+# Timestamp
+
+**Timestamp** digunakan dalam dunia [modding](/wiki/Modding) untuk merujuk pada suatu waktu atau [hit object](/wiki/Hit_Objects) pada suatu [beatmap](/wiki/Beatmaps). Saat diposting di [diskusi beatmap](/wiki/Beatmap_Discussion), timestamp mempunyai bentuk berupa tautan yang dapat digunakan untuk melompat ke titik waktu dari timestamp tersebut di [editor](/wiki/Beatmap_Editor).
+
+menyalin (`Ctrl` + `C`) sebuah hit object yang dipilih di editor akan menyertakan timestamp ke clipboard. Waktu yang berada di kiri bawah juga dapat diklik dan disalin.
+
+## Format
+
+Sintaks timestamp: `menit:detik:milidetik (angka kombo)`
+Sintaks link: `osu://edit/timestamp`
```

Comment:
I'd say to keep using `Syntax` as `Sintaks` isn't an actual word in KBBI. And due to it's common nature I believe `Syntax` is easier to understand rather than `Sintaks`.

Reply:
should it be changed into `Timestamp syntax`? or, just change `Sintaks` to `Syntax` (it will be `Syntax timestamp`, same as en.md)

Reply:
aight, changed.